pkg/psalabelsyncer/podsecurity_label_sync_controller_test.go

```diff
@@ -362,18 +405,21 @@ func TestEnforcingPodSecurityAdmissionLabelSynchronizationController_sync(t *tes
 			"controlled-namespace-previous-enforce-labels/testspecificsa3":                  sets.NewString("scc_restricted"),
 			"controlled-namespace-previous-warn-labels/testspecificsa3":                     sets.NewString("scc_restricted"),
 			"controlled-namespace-previous-enforce-version-different-owner/testspecificsa3": sets.NewString("scc_restricted"),
+			"controlled-namespace-different-owner-and-sync-label-true/testspecificsa3":      sets.NewString("scc_restricted"),
+			"controlled-namespace-different-owner-and-sync-label-unset/testspecificsa3":     sets.NewString("scc_restricted"),
```

Comment:
I have not enough insights into the test cases, but don't we need to list: `non-controlled-namespace-different-owner-and-sync-label-false`?

Reply:
No we don't need to. As this namespace is non-controlled, we don't need a service account fixture.

Reply:
Thx for the explanation.